PasswordGenerator.py

```diff
@@ -0,0 +1,29 @@
+import random
+import string
+
+def randomPassGenerator(length,supporting=''):
+    password=str(supporting)
+    support_len=len(str(supporting))
+    random_len=length-support_len
+
+    for i in range(random_len):
+        word=random.choice([random.choice(string.ascii_letters),random.choice(string.punctuation)])

```

Comment:
You have to ask user if they want to include alphabets/numeric/special-characters.